examples/experimental/dagster-airlift/dagster_airlift_tests/airflow_project/dags/simple_dag.py

```diff
@@ -1,7 +1,7 @@
 from datetime import datetime
 
 from airflow import DAG
-from airflow.operators.python_operator import PythonOperator
```

Comment:
These are deprecated import paths that are only supported dynamically (thus illegible to type checkers) in airflow 2.0+. I changed them to the new canonical paths supported from  airflow 2.0 and on.